client/webserver/locales/parse_test.go

```diff
@@ -0,0 +1,102 @@
+package locales
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
+func TestTokens(t *testing.T) {
+	tmplFile := []byte(`
+{{define "settings"}}
+{{template "top" .}}
+{{$passwordIsCached := .UserInfo.PasswordIsCached}}
+<div id="main" data-handler="settings" class="text-center py-5 overflow-y-auto">
+  <div class="settings">
+    <div class="form-check">
+      <input class="form-check-input" type="checkbox" value="" id="showPokes"{{if .UserInfo.ShowPopups}} checked{{end}}>
+      <label class="form-check-label" for="showPokes">
+        [[[Show pop-up notifications]]]
+      </label>
```

Comment:
For this test, the only necessary HTML elements we need are those with `[[[whatever-token-here]]]`, the rest can be trimmed off, I think.

Reply:
Well, the reason why I want to leave some HTML code in is to make sure this pattern doesn't catch any extra HTML code that it shouldn't,

I really see no benefit to trimming it down even further